pyscript.core/src/core.css

```diff
@@ -42,3 +42,34 @@ mpy-config {
 .mpy-editor-run-button:disabled {
     opacity: 1;
 }
+
+@keyframes spinner {
+    to {
+        transform: rotate(360deg);
+    }
+}
+.py-editor-run-button:disabled *,
+.mpy-editor-run-button:disabled * {
+    display: none; /* hide all the child elements of the run button when it is disabled */
```

Comment:
nit: if that's the purpose, technically ` > *` instead of ` *` would be better, as only direct child needs to be hidden

Reply:
Added in commit 142c55b137e4aa0309b819ca19a79860cdec5dfb and it still works as intended.